.github/workflows/docker-x86.yml

```
@@ -0,0 +1,74 @@
+name: Build and test x86 Docker Images
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/docker-x86.yml"
+
+      - "all-spark-notebook/**"
+      - "base-notebook/**"
+      - "datascience-notebook/**"
+      - "minimal-notebook/**"
+      - "pyspark-notebook/**"
+      - "r-notebook/**"
+      - "scipy-notebook/**"
+      - "tensorflow-notebook/**"
+
+      - "tagging/**"
+      - "test/**"
+      - "conftest.py"
+      - "Makefile"
+      - "pytest.ini"
+      - "requirements-dev.txt"
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - ".github/workflows/docker-x86.yml"
+
+      - "all-spark-notebook/**"
+      - "base-notebook/**"
+      - "datascience-notebook/**"
+      - "minimal-notebook/**"
+      - "pyspark-notebook/**"
+      - "r-notebook/**"
+      - "scipy-notebook/**"
+      - "tensorflow-notebook/**"
+
+      - "tagging/**"
+      - "test/**"
+      - "conftest.py"
+      - "Makefile"
+      - "pytest.ini"
+      - "requirements-dev.txt"
```

Allows us to run this manually, which is very useful to verify if something has broken due to a PR or due to a change over time.

```suggestion
      - "requirements-dev.txt"
  workflow_dispatch:
```

---

Nice idea!
Maybe it's worth adding to other workflows as well.